udvarlas.pl

```diff
@@ -45,7 +45,6 @@
 		"annyira akarnám",
 		"annyira imádnám",
 		"úgy vágynám",
-		"vállalnám én",
```

Comment:
Ezek csak akkor hangzanak jól, ha együtt jönnek ki. Máskülönben: "nagyon kedvem lenne a halált is érted", "vállanám én az érzést elmerülni szemeid tavában", stb...